<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="obstacle_4">
    <static>true</static>
    <link name="link">
      <pose>0 0 0 0 0 0</pose>
      <collision name="collision">
        <geometry>
          <cylinder>
              <radius>0.2</radius>
              <length>5.0</length>
          </cylinder>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <cylinder>
              <radius>0.2</radius>
              <length>5.0</length>
          </cylinder>
        </geometry>        
      </visual>
    </link>
  </model>
</sdf>
